six/modules/c++/six.sidd/include/six/sidd/Annotations.h

```diff
@@ -35,6 +35,18 @@ struct Annotation
     std::string identifier;
     mem::ScopedCopyablePtr<SFAReferenceSystem> spatialReferenceSystem;
     std::vector<mem::ScopedCloneablePtr<SFAGeometry> > objects;
+
+    bool operator==(const Annotation& rhs) const

```

Comment:
Unrelated but again as long as you're in here... `Annotation` does not need a constructor since all that stuff will default to what it's set to anyway, so go ahead and remove it.